src/stories/atoms/Shape/Shape.stories.tsx

```diff
@@ -0,0 +1,33 @@
+// Global Imports
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+// Local Imports
+import { Shape } from 'stories/atoms/Shape/Shape';
+import { defaultTheme } from 'themes';
+
+export default {
+  title: 'Design System/Atoms/Shape',
+  component: Shape,
+} as ComponentMeta<typeof Shape>;
+
+const Template: ComponentStory<typeof Shape> = (args) => <Shape {...args} />;
+
+export const Default = Template.bind({});
+Default.args = {
+  backgroundColor: defaultTheme.primaryGray,
+  shapeHeight: '10rem',
```

Comment:
I appreciate the changes you made for the sake of string concatenation, but your new approach opens up some other issues. 

_What happens when someone passes a value that does not result in valid CSS?_ 

```ts
// This is valid typescript/tsx but will not produce valid CSS
<Shape ...rest shapeHeight='10remm' />
```

Reply:
For this it seems I am weighing two paths. The first path is I use strings for the `shapeWidth` and `shapeHeight` props. If I chose strings, I gain the advantage of a component that can accept a variety of units for `width:` and `height:`. The second path is use numbers for `shapeWidth` and `shapeHeight` and interpolate those numbers into the styled-component CSS. With numbers I lose the potential variety of units, but I lower the risk for error. Ultimately, in weighing the two paths, I would say that the the benefits gained from unit variety outweigh the risk of potential coding errors. I believe this because if a programmer were to make an error and submit invalid CSS, they would see the discrepancy in Storybook. I believe a programmer would prefer the flexibility of units and re-usability over added error prevention. Hahaha now I could feel this way because I am missing something conceptually here. Particularly in regards to how these components come to ultimately be used in the application